clarifai/client/app.py

```diff
@@ -182,6 +183,31 @@ def list_installed_module_versions(self, filter_by: Dict[str, Any] = {}) -> List
         for imv_info in all_imv_infos
     ]
 
+  def list_runners(self, filter_by: Dict[str, Any] = {}) -> List[Runner]:
+    """List all runners for the app
+
+    Args:
+        filter_by (dict): A dictionary of filters to apply to the list of runners.
+
+    Returns:
+        List[Runner]: A list of Runner objects for the runners in the app.
+
+    Example:
+        >>> from clarifai.client.app import App
+        >>> app = App(app_id="app_id", user_id="user_id")
```

Comment:
runners are actually not owned apps like other resources, they are owned by the user/org (like a sibling to apps). This is how github does it and in the future we can add runner groups like github has to limits runners to certain apps if needed. So for the API client the runners should be off the User object instead of App.

Reply:
Got it. Now changed to User object instead of App.